ts/features/payments/checkout/utils/index.ts

```diff
@@ -81,3 +81,9 @@ export const getPaymentPhaseFromStep = (
       return "pagamento";
   }
 };
+
+/**
+ * This function checks if a date is a dumb date
+ * that some entities use to represent a missing date.
+ */
+export const isPaymentDumbDate = (date: string) => date === "31/12/2099";
```

Comment:
Are we sure that this is the only "dumb" date we get? I mean. This is the only "dumb" date scenario, or we could have some others patterns/formats. Is there a better (read: more maintainable in the long run way) to handle this?

Reply:
We're going to improve pagoPA specs to better explain entities shouldn't use any dumb date, as the field is not mandatory. But it's not going to be effective in the short run 💔

Reply:
@shadowsheep1 I agree with you. Even if actually `31/12/2099` is the only "dumb date" found, I set up this function to be more maintainable in the future in case we identify other dates. Thanks for the suggestion!

Addressed into 448e7c310d0fd3b67af92e6141222c4be5bcebcc